lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.py

```diff
@@ -77,3 +104,11 @@ def getPrimPath(ufePathStr):
     ufePath = ufe.PathString.path(ufePathStr)
     parentPath = ufePath.segments[1]
     return str(parentPath)
+
+def getUniqueMayaReferencePrimName(ufePathStr, startName=None):
+    '''Helper function to get a unique name for the Maya Reference prim.'''
+    newPrimName = kDefaultMayaReferencePrimName if not startName else startName
+    prim = mayaUsd.ufe.ufePathToPrim(ufePathStr)
+    if prim.IsValid():
+        return mayaUsd.ufe.uniqueChildName(prim, newPrimName)
+    return newPrimName
```

Comment:
Called from the MEL file to make sure the text entered in prim name field is unique.